tests/tests/src/lampo_tests.rs

```diff
@@ -251,7 +251,7 @@ pub async fn decode_invoice() -> error::Result<()> {
 
     log::info!(target: &node2.info.node_id, "invoice generated `{:?}`", invoice);
 
-    let decode: response::Bolt11InvoiceInfo = node2
+    let decode_result: response::DecodeResult = node2
```

Comment:
call the type `Decode` we already know that is the result because it is under the `response::` module